### PR TITLE
[CL-58] Setup Database

### PR DESCRIPTION
**Pull request by jeepies**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#### Setting up the database
+> [!WARNING]  
+> Please ensure that the `setup-db.sh` file is executable. On Linux, `chmod a+x ./setup-db.sh` will work!
+
+1. Ensure that Docker is started
```
Comment (stevelw) [inline README.md:5]: How do we start Docker? I can't see this in the dependencies or scripts.

**Review by stevelw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jeepies) [reply README.md:5]: Starting the container is documented, starting Docker is up to how you've installed it and based on OS.

Comment (jeepies) [reply README.md:5]: Further documented the setup process in CL-59

Comment (PhilTBatt) [reply README.md:5]: Hello

**Review by stevelw — APPROVED** [review]

**Pull request merged**